### ISSUE TITLE
[0.9.6][DATAGRIP 2024.2.2] java.lang.NoClassDefFoundError: org/jetbrains/plugins/terminal/TerminalToolWindowManager

### ISSUE THREAD

**Issue by Jesusmcyc** [Codium-ai/codiumai-jetbrains-release#323]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
java.lang.NoClassDefFoundError: org/jetbrains/plugins/terminal/TerminalToolWindowManager
	at ai.codium.ui.actions.AddTerminalSelectionAsContextAction$Companion.addActionToOldTerminal(AddTerminalSelectionAsContextAction.kt:70)
	at ai.codium.CodiumInitialStartupActivity.runActivity(CodiumInitialStartupActivity.kt:24)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runOldActivity(StartupManagerImpl.kt:340)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runOldActivity(StartupManagerImpl.kt:69)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$6$1.invoke(StartupManagerImpl.kt:291)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$6$1.invoke(StartupManagerImpl.kt:290)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContextInner(coroutines.kt:339)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invokeSuspend(coroutines.kt:232)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at com.intellij.openapi.progress.CoroutinesKt$blockingContext$2.invoke(coroutines.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:261)
	at com.intellij.openapi.progress.CoroutinesKt.blockingContext(coroutines.kt:231)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.kt:290)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runPostStartupActivities(StartupManagerImpl.kt:69)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$3$2.invokeSuspend(StartupManagerImpl.kt:193)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$3$2.invoke(StartupManagerImpl.kt)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$3$2.invoke(StartupManagerImpl.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:163)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.ide.startup.impl.StartupManagerImpl$runPostStartupActivities$3.invokeSuspend(StartupManagerImpl.kt:192)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
Caused by: java.lang.ClassNotFoundException: org.jetbrains.plugins.terminal.TerminalToolWindowManager PluginClassLoader(plugin=PluginDescriptor(name=Codiumate - Code, test and review with confidence - by CodiumAI, id=codiumai.codiumai, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\DataGrip2024.2\plugins\codium-ai-plugin, version=0.9.6, package=null, isBundled=false), packagePrefix=null, state=active, parents=PluginDescriptor(name=Git, id=Git4Idea, moduleName=intellij.vcs.git/terminal, descriptorPath=intellij.vcs.git.terminal.xml, path=D:\- Install\JetBrains\DataGrip\plugins\vcs-git, version=242.21829.162, package=git4idea.terminal, isBundled=true), PluginDescriptor(name=Git, id=Git4Idea, moduleName=intellij.vcs.git/newUiOnboarding, descriptorPath=intellij.vcs.git.newUiOnboarding.xml, path=D:\- Install\JetBrains\DataGrip\plugins\vcs-git, version=242.21829.162, package=git4idea.newUiOnboarding, isBundled=true), PluginDescriptor(name=Git, id=Git4Idea, moduleName=intellij.vcs.git.coverage, descriptorPath=intellij.vcs.git.coverage.xml, path=D:\- Install\JetBrains\DataGrip\plugins\vcs-git, version=242.21829.162, package=com.intellij.vcs.git.coverage, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.impl, descriptorPath=intellij.platform.vcs.impl.xml, path=D:\- Install\JetBrains\DataGrip\lib, version=242.21829.162, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.log.impl, descriptorPath=intellij.platform.vcs.log.impl.xml, path=D:\- Install\JetBrains\DataGrip\lib, version=242.21829.162, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.dvcs.impl, descriptorPath=intellij.platform.vcs.dvcs.impl.xml, path=D:\- Install\JetBrains\DataGrip\lib, version=242.21829.162, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.collaborationTools, descriptorPath=intellij.platform.collaborationTools.xml, path=D:\- Install\JetBrains\DataGrip\lib, version=242.21829.162, package=null, isBundled=true), PluginDescriptor(name=Git, id=Git4Idea, descriptorPath=plugin.xml, path=D:\- Install\JetBrains\DataGrip\plugins\vcs-git, version=242.21829.162, package=git4idea, isBundled=true), )
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:157)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 28 more

```

# Other Information
```
=== About ===
Build version: DataGrip 2024.2.2 Build: #DB-242.21829.162 August 29, 2024
Theme: Dark
JRE: 21.0.3+13-b509.11, JetBrains s.r.o.
JVM: 21.0.3+13-b509.11, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 11.0 (amd64)

=== Plugins ===
Custom plugins: [.env files support (2024.2.1), SciaReto Mind Map (1.6.8), JetBrains AI Assistant (242.21829.196), Rainbow Brackets (2024.2.6-241), Big Data Tools Core (242.21829.142), Big Data File Viewer (242.20224.159), Databricks (242.20224.419), Remote File Systems (242.21829.142), IdeaVim (2.16.0), Metastore Core (242.20224.159), Codiumate - Code, test and review with confidence - by CodiumAI (0.9.6), Spark (242.20224.159), Zeppelin (242.20224.159), Kafka (242.20224.159), Flink (242.20224.159), Big Data Tools (242.20224.159)]
Disabled plugins:[]
```

**Comment by ArtsiomCh** [comment]
Thanks for reporting!
Work is in progress. Fix should be available tomorrow.
Plugin functionality should not be affecting, just ignore that exception.

**Comment by ArtsiomCh** [comment]
Fixed in 0.9.7
